src/Http/Download.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Http;
+
+use OpenEuropa\CdtClient\Contract\RestInterface;
+use OpenEuropa\CdtClient\Contract\TokenAwareInterface;
+use OpenEuropa\CdtClient\Exception\InvalidStatusCodeException;
+use OpenEuropa\CdtClient\Traits\SerializerAwareTrait;
+use OpenEuropa\CdtClient\Traits\TokenAwareTrait;
+use OpenEuropa\CdtClient\Traits\ValidationAwareTrait;
+
+/**
+* A class that download files from the API.
+ *
+ * @see TokenAwareInterface
+ * @see ValidationAwareTrait
+ * @see
+*/
+class Download implements TokenAwareInterface
+{
+    use TokenAwareTrait;
+    use ValidationAwareTrait;
+    use SerializerAwareTrait;
+
+    public function __construct(protected RestInterface $rest)
+    {
+    }
+
+    /**
+     * @param array<string, string> $headers
+     */
+    public function downloadFile(string $uri, array $headers = []): string
```

Comment:
$headers is not used

Reply:
Probably an unnoticed leftover of the rebase, sorry for that